### PR TITLE
Update Excel.XlErrorChecks.md

### PR DESCRIPTION
**Pull request by PrashSMSFT**
Added a new error object xlStaleValue with Value 12 and Description "The cell contains an uncalculated value."

### CODE REVIEW

File: api/Excel.XlErrorChecks.md [hunk 1]
```diff
@@ -23,5 +23,6 @@ Specifies the type of error object to be retrieved from the **Errors** collectio
 | **xlOmittedCells**|5|Cells omitted.|
 | **xlTextDate**|2|Date entered as text.|
 | **xlUnlockedFormulaCells**|6|Formula cells are unlocked.|
+| **xlStaleValue**|12|The cell contains an uncalculated value.|
```
Comment (lindalu-MSFT) [inline api/Excel.XlErrorChecks.md:26]:
```suggestion
```

File: api/Excel.XlErrorChecks.md [hunk 2]
```diff
@@ -24,4 +24,4 @@ Specifies the type of error object to be retrieved from the **Errors** collectio
 | **xlTextDate**|2|Date entered as text.|
```
Comment (lindalu-MSFT) [inline api/Excel.XlErrorChecks.md:24]:
```suggestion
| **xlStaleValue**|12|The cell contains an uncalculated value. |
| **xlTextDate**|2|Date entered as text. |
```

**Review by lindalu-MSFT — APPROVED** [review]
Approved after I alphabetized the list. Thanks for your help keeping these docs up to date!

**Comment by PrashSMSFT** [comment]
[like]  Prash Shirolkar reacted to your message:
________________________________

**Pull request merged**